.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,62 @@
+name: Pre-Commit
```

Comment:
Please replace the content of this file with this one - https://github.com/terraform-aws-modules/terraform-aws-lambda/blob/master/.github/workflows/pre-commit.yml